src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  background-color: #e5e5e5;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+}
+
+.moyo-logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+nav {
+  display: block;
+}
```

Comment:
1) you should avoid use tag selectors for styling
2) nav tag is a block by default
```suggestion
```